src/cdk/stepper/stepper-button.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {Directive} from '@angular/core';
+import {CdkStepper} from './stepper';
+
+@Directive({
+  selector: 'button[cdkStepperNext]',
+  host: {
+    '(click)': '_onClick()',
```

Comment:
I don't quite follow what you mean; even without the click handler being inherited, you can still directly have `host: {'(click)': '_stepper.next()'}`